### PR TITLE
feat(FOROME-918): implemented functional units selector

### PR DESCRIPTION
**Pull request by AlMaQntr**
https://quantori.atlassian.net/browse/FOROME-918

with units list refactoring

### CODE REVIEW

File: src/components/fn-label.tsx [hunk 1]
```diff
@@ -1,34 +1,26 @@
 import { ReactElement } from 'react'
 import cn, { Argument } from 'classnames'
-import { observer } from 'mobx-react-lite'
 
 import { t } from '@i18n'
-import { IStepData } from '@store/dtree'
 
 interface IProps {
```
Comment (Lapk1n) [inline src/components/fn-label.tsx]: IFnLabelProps

**Comment by Lapk1n** [comment]
Screenshot 2022-05-19 at 10 32 10
Is it correct behaviour to show added func attrs in left side of FR, but not show them on the same side of Dtree?

**Comment by AlMaQntr** [comment]
> Screenshot 2022-05-19 at 10 32 10
> 
> Is it correct behaviour to show added func attrs in left side of FR, but not show them on the same side of Dtree?

Yes, it's correct

File: src/components/fn-label.tsx (hunk 1 above)
Comment (AlMaQntr) [reply src/components/fn-label.tsx]: fixed, thx

**Review by AndreyUstyumenko — APPROVED** [review]

**Review by Lapk1n — APPROVED** [review]

**Pull request merged**